### DISCUSSION TITLE
GitHub CoPilot

### DISCUSSION THREAD

**Discussion by TanyaNijhawan2004** [community/community#30685] [Copilot Conversations]
I have student developer pack which is approved but still when i try to enable my copilot it says

GitHub Copilot could not connect to server. Extension activation failed: "certificate has expired"

What should i so ?

**Comment by davecheney** [comment]
@TanyaNijhawan2004 Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?
